www/src/components/blog-post-preview-item.js

```diff
@@ -56,15 +70,14 @@ class BlogPostPreviewItem extends React.Component {
               <Link
                 to={post.frontmatter.author.fields.slug}
                 css={{
-                  boxShadow: `none !important`,
-                  borderBottom: `0 !important`,
+                  color: `${colors.gatsby} !important`,
+                  fontSize: `102%`,
```

Comment:
We don't need to increase font-size here because we don't mix Futura with Spectral.